components/script/textinput.rs

```diff
@@ -510,6 +609,24 @@ impl<T: ClipboardProvider> TextInput<T> {
                 self.delete_char(Direction::Backward);
                 KeyReaction::DispatchInput
             }
+            #[cfg(target_os = "macos")]
+            (None, Key::Left) if mods.contains(SUPER) => {
+                self.adjust_horizontal_to_end(Direction::Backward, maybe_select);
+                KeyReaction::RedrawSelection
+            }
+            #[cfg(target_os = "macos")]
+            (None, Key::Right) if mods.contains(SUPER) => {
+                self.adjust_horizontal_to_end(Direction::Forward, maybe_select);
+                KeyReaction::RedrawSelection
+            }
+            (None, Key::Left) if mods.contains(ALT) => {
+                self.adjust_horizontal_by_word(Direction::Backward, maybe_select);
+                KeyReaction::RedrawSelection
+            }
+            (None, Key::Right) if mods.contains(ALT) => {
+                self.adjust_horizontal_by_word(Direction::Forward, maybe_select);
+                KeyReaction::RedrawSelection
+            }
```

Comment:
Should ALT-Left and ALT-Right be forward and backward traversal of the session history?